src/pages/Signin/Signin.js

```diff
@@ -27,6 +26,24 @@ const Signin = () => {
       /^(?=.*[a-zA-z])(?=.*[0-9])(?=.*[$`~!@$!%*#^?&\\(\\)\-_=+]).{8,16}$/
     );
 
+  const loginFetch = () => {
+    fetch('http://10.58.52.91:3000/users/signin', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        account: idValue,
+        password: pwValue,
+      }),
+    })
+      .then(res => res.json())
+      .then(token => {
+        console.log(token);
+        localStorage.setItem('token', token.token);
+        navigate('/');
+      });
```

Comment:
```suggestion
      .then(loginData => {
        localStorage.setItem('token', loginData.token);
        navigate('/');
      });
```
로그인에 성공하면 토큰이 발급되지만,
로그인에 성공하지않으면 토큰이 없어요.

지금은 무조건 로컬스토리지에 저정하게 되어있는데,
토큰이 있을 경우에만 저장하고,
없을땐 유저한테 로그인이 안됐다고 알려주는 코드로 수정 해 보세요.